exercises/e001/results/propiro/outputBase32faster.py

```diff
@@ -0,0 +1,32 @@
+import string
+import random
+import argparse
+
+parser = argparse.ArgumentParser(description='base32 key generator script')
+parser.add_argument('-s','--size', const=32, default=32, type=int, help='size of key, default=32', nargs='?')
```

Comment:
Can you make it an optional argument? As in (I didn't run your command yet, just by reading): `cmd.py -s 15` versus `cmd.py 15`